Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -358,17 +366,25 @@ void Mandelbrot::resize_event(GUI::ResizeEvent& event)
     m_set.resize(event.size());
 }
 
-void Mandelbrot::export_image(String const& export_path)
+void Mandelbrot::export_image(String const& export_path, ImageType image_type)
 {
     m_set.resize(Gfx::IntSize { 1920, 1080 });
-    auto png = Gfx::PNGWriter::encode(m_set.bitmap());
+    ByteBuffer encoded_data;
+    if (image_type == ImageType::BMP) {
```

Comment:
If we make this a switch statement instead, we can ensure it gets updated when a new image type is added as it will result in a compile error for unhandled cases